packages/webapp/src/apiConfig.js

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2019-2022 LiteFarm.org
+ *  Copyright 2019-2024 LiteFarm.org
```

Comment:
Does the new copyright standard just use the one current year now? https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1269202970/Copyright+GNU+General+Public+License

Reply:
Well I have been doing it wrong then lol -- I remember Ivan sharing some kind of blog saying 1 year was enough. I think I have actually deleted these intervals in favour of 1 year approach.

Reply:
Oh no 😂  The original block we were copy-pasting had all the years that LiteFarm was active, and I think Ivan was just asking us to get rid of all the other dates if the file itself hadn't existed back then. I think this method with the range of dates (from creation up to last modification) is pretty standard.